.craft.yml

```diff
@@ -157,6 +157,24 @@ targets:
           - nodejs20.x
     license: MIT
 
+  # NOTE: We publish the v8 layer under its own name so people on v8 can still get patches
+  # whenever we release a new v8 version—otherwise we would overwrite the current major lambda layer.
+
+  # AWS Lambda Layer target
+  - name: aws-lambda-layer
+    includeNames: /^sentry-node-serverless-\d+.\d+.\d+(-(beta|alpha)\.\d+)?\.zip$/
```

Comment:
No -rc like above?

Reply:
Good catch, will add. Though I'm not sure we'll have rc's for it but it can't hurt.